dll/cpl/desk/lang/ru-RU.rc

```diff
@@ -179,7 +179,7 @@ CAPTION "Общие"
 FONT 8, "MS Shell Dlg"
 BEGIN
     GROUPBOX "Экран", -1, 7, 6, 237, 56, WS_GROUP
-    LTEXT "Размер шрифта:", -1, 14, 20, 222, 9//FIXME: add accel
+    LTEXT "Размер шрифта:", -1, 14, 20, 222, 9//FIXME: add accel. I used "&Размер шрифта:" for the backport
```

Comment:
Windows XP ru-RU has different word here (Масштаб), but let's use the same mnemonic key:
```suggestion
    LTEXT "Размер &шрифта:", -1, 14, 20, 222, 9//FIXME: I used "&Размер шрифта:" for the backport
```
![image](https://github.com/user-attachments/assets/2295600c-8929-40f6-b51f-850f5b213e68)

Reply:
I might do that, but we do have 2 "&Размер шрифта:" in this rc file. In my backport solution they can both currently benefit from string pooling. If you suggest "Размер &шрифта:" here now, I would like you to check beforehand, whether the same thing with the same accel could also be used for line66:
    LTEXT "&Размер шрифта:", IDC_STATIC, 7, 184, 104, 9

Is that also used in the russian XPSP3 **there** as     LTEXT "Размер &шрифта:", IDC_STATIC, 7, 184, 104, 9
???

Reply:
Here: ![image](https://github.com/user-attachments/assets/d450ab79-02b1-4d52-aea4-5be0781e4f41)

Reply:
Okay, I will apply your suggestion then only at the single place, and only within master. But I will not port that suggestion back into older branches.

Reply:
> In my backport solution they can both currently benefit from string pooling.

I would suggest you actually verify with both MSVC and GCC that string pooling is a thing (or is not) for resources.
(Because string pooling is a compiler thing, not a resource one.....)

Reply:
It is definitely not effective yet. Not needed to test that. I am sure about that. But it might be a thing in 10 years.

Reply:
> But it might be a thing in 10 years.

Not really. Because how resource strings are stored and indexed in the binary file.

Reply:
Then call it performed to optimize "consistency for improved users experience".

Reply:
Resolved. I keep this uncollapsed to improve visibility of Stasms pic.